german/Korallen/Hoehle/Unterwasserstation/Person/GespraechMitHoehlenmann/Gespraech.md

```diff
@@ -0,0 +1,2 @@
+Der Mann jault laut auf und geht auf sie los, in letzter Sekunde entsinnen Sie sich Ihrer fast vergessenen Judokenntnisse und legen ihn mit einem gekonnten Wurf aufs Kreuz.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.